### DISCUSSION TITLE
Cypress-cucumber-preprocessor DRY RUN option

### DISCUSSION THREAD

**Discussion by cyrilgeorge153** [badeball/cypress-cucumber-preprocessor#1213] [General]
Does Cypress-cucumber-preprocessor has a DRY RUN option, so that step defintions will be printed into console. Later user can copy it to stepdefinition.js files

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/diagnostics.md
